index.html

```diff
@@ -34,6 +34,12 @@
              company: "Block",
              companyURL: "https://www.tbd.website",
              w3cid: 116851
+           }, {
+             name: 'Mike Prorock',
+             url: 'https://github.com/mprorock',
+             company: 'mesur.io',
+             companyURL: 'https://mesur.io/',
+             w3cid: 109171
```

Comment:
Is this correct? Our database refers to 130636, see https://www.w3.org/users/130636/

Reply:
good catch, thanks